server/src/test/java/org/spine3/server/storage/EventStorageShould.java

```diff
@@ -378,11 +375,11 @@ private void givenSequentialRecords() {
     private void givenSequentialRecords(long deltaSeconds, int deltaNanos) {
         final Duration delta = Duration.newBuilder().setSeconds(deltaSeconds).setNanos(deltaNanos).build();
         time1 = getCurrentTime().toBuilder().setNanos(POSITIVE_DELTA * 10).build(); // to be sure that nanos are bigger than delta
-        record1 = projectCreated(time1);
+        record1 = Given.EventStorage.projectCreated(time1);
```

Comment:
you are retrieving not `EventStorage` but `EventStorageRecord`, so rename this class (the same thing for other classes creating records, etc).